clearwater-memcached/usr/share/clearwater/infrastructure/conf/memcached_11211.monit

```diff
@@ -46,11 +46,11 @@ check process memcached_process with pidfile "/var/run/memcached_11211.pid"
   if memory > 80% for 6 cycles then restart
 
 # Clear any alarms if the process has been running long enough.
-check program memcached_uptime with path "/usr/share/clearwater/bin/check-uptime /var/run/memcached_11211.pid"
+check program memcached_uptime with path "/usr/share/clearwater/bin/check-uptime /var/run/memcached_11211.pid monit 3500.1"
   group memcached
   depends on memcached_process
   every 3 cycles
-  if status = 0 then exec "/usr/share/clearwater/bin/issue_alarm.py monit 3500.1"
+  if status != 0 then alert
```

Comment:
We want to mark the uptime check as failed when it fails. As I understand it, that's what alert does.